docs/how-to/pin-files.md

```diff
@@ -8,6 +8,10 @@ description: Learn how to pin files in IPFS in order to keep your files and othe
 
 Pinning is a very important concept in IPFS. IPFS semantics try to make it feel like every single object is local — there is no "retrieve this file for me from a remote server", just `ipfs cat` or `ipfs get`, which act the same way no matter where the actual object is located.
 
+:::tip Web3.Storage
```

Comment:
would kill this

Reply:
Removed in https://github.com/ipfs/ipfs-docs/pull/847/commits/1706182bc096083d92bcf3bd3fc1c63d2a5aba03.